table-api/src/main/java/io/deephaven/db/tables/utils/DBNameValidator.java

```diff
@@ -199,48 +169,55 @@ public static String legalizeColumnName(String name, Function<String, String> re
     }
 
     /**
-     *  Attempts to return a legal name based on the passed in {@code name}.
+     * Attempts to return a legal name based on the passed in {@code name}.
      *
-     *  Illegal characters are simply removed. Custom replacement is possible through {@code customReplace}
+     * Illegal characters are simply removed. Custom replacement is possible through
+     * {@code customReplace}
      *
-     *  To avoid duplicated names, anything in the set {@code takenNames} will not be returned. Theses duplicates
-     *  are resolved by adding sequential digits at the end of the variable name.
+     * To avoid duplicated names, anything in the set {@code takenNames} will not be returned.
+     * Theses duplicates are resolved by adding sequential digits at the end of the variable name.
```

Comment:
`s|Theses|These|`

Also, newlines in javadoc mean ~nothing (afaik).

Use `<p>` or `<br>` for line breaks.  Maybe our javadoclet is more competent than the ugly html ones I remember...